src/stores/discord.ts

```diff
@@ -29,6 +30,12 @@ export function getFormattedDate(date: string, withTime = false) {
 	return new Date(date).toLocaleDateString(navigator.language, options);
 }
 
+export function getUserName(user: IDiscordUser | User) {
+	return user.discriminator === '0'
```

Comment:
In light of the username change, discriminators should become optional in prisma, and store the new system as NULL. This solution checks for a 0, which honestly shouldn't be there to begin with.

Reply:
> In light of the username change, discriminators should become optional in prisma, and store the new system as NULL. This solution checks for a 0, which honestly shouldn't be there to begin with.

ATM discord actually uses 0 as discriminators, who knows what they do if the new system is fully run out but for the time being I think this is the right way over null check. Since technically speaking it's 0 not null.

Thank discord for this jank of an update.

Reply:
I'd still rather have it nullable, in case they do throw the discriminator out entirely. I can see them doing so.

Reply:
They won't remove the discriminator since it breaks their clients as well. Especially since there's millions of users that don't change their username. 
I only see them removing discrim's in a future api version.
I'll see about making it nullable